onnxruntime/test/providers/cpu/tensor/pad_test.cc

```diff
@@ -809,22 +809,22 @@ TEST(PadOpTest, ConstantPadAxes) {
   OpTester test("Pad", 18);
   test.AddAttribute("mode", "constant");
   test.AddInput<int32_t>("data", {1, 2, 2, 2},
-  {
-    1, 1,
-    1, 1,
-    1, 1,
-    1, 1});
+                         {1, 1,
+                          1, 1,
+                          1, 1,
+                          1, 1});
   test.AddInput<int64_t>("pads", {4}, {0, 1, 0, 1});
   test.AddInput<int32_t>("value", {1}, {0});
+#if defined(USE_COREML) && defined(__APPLE__)
+  test.AddInput<int64_t>("axes", {2}, {1, 3}, true /*axes_is_initializer*/);
```

Comment:
is this the only Pad test which sets the "axes" input? should we add some more tests, at least to cover the case where "axes" is set and the last two dimensions are padded?

Reply:
yes... we really need to add additional test cases here. will do.